dissect/target/plugins/browsers/chromium.py

```diff
@@ -0,0 +1,157 @@
+from typing import Iterator
+
+from flow.record import Record
+
+from dissect.sql import sqlite3
+from dissect.sql.exceptions import Error as SQLError
+from dissect.sql.sqlite3 import SQLite3
+from dissect.util.ts import webkittimestamp
+
+from dissect.target.helpers.fsutil import TargetPath
+from dissect.target.exceptions import FileNotFoundError, UnsupportedPluginError
+from dissect.target.helpers.descriptor_extensions import UserRecordDescriptorExtension
+from dissect.target.helpers.record import create_extended_descriptor
+from dissect.target.plugin import Plugin, export
+from dissect.target.plugins.browsers.browser import (
+    GENERIC_HISTORY_RECORD_FIELDS,
+    try_idna,
+)
+
+
+class ChromiumMixin:
+    """Mixin class with methods for Chromium-based browsers."""
+
```

Comment:
```suggestion
    DIRS = []
    HISTORY_RECORD = create_extended_descriptor([UserRecordDescriptorExtension])(
        "browser/chromium/history", GENERIC_HISTORY_RECORD_FIELDS
    )
```
Good idea to add some sane default values in the mixin itself.